Simplified/Utilities/UI/NYPLActivityIndicatorMessageViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  NYPLActivityIndicatorMessageViewController.swift
+//  SimplyE
+//
+//  Created by Ernest Fan on 2022-06-06.
+//  Copyright © 2022 NYPL. All rights reserved.
+//
+
+import UIKit
+
+/// Display an activity indicator with a message in the center of the screen,
+/// with a dimmed transparent background.
+class NYPLActivityIndicatorMessageViewController: UIViewController {
+  private let spacing: CGFloat = 20.0
+  
+  // MARK: - Life Cycle
+  
+  init(with message: String) {
```

Comment:
in initializers "with" is typically not used by convention. You can just do `init(message: String)`